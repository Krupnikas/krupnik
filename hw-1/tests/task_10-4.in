7 123 44.44445
