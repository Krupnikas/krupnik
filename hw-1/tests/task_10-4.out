Char: {
Int: 7
Real: 44.444450
